lib/Mojolicious/resources/templates/mojo/menubar.html.ep

```diff
@@ -86,6 +150,11 @@
       $('html, body').animate({scrollTop: offset.top - mojobarHeight}, 1);
     }
   }
+  document.querySelector("#mojobar-brand").addEventListener("click", function (e) {
+    if (window.innerWidth > 900) return;
```

Comment:
I don't like the if without block, is that really considered good JavaScript style?

Reply:
I'm not sure which JavaScript style we're going for, but I don't mind this style, in the same way as I like `return unless $whatever` in perl.

Nevertheless, I'll change it in a new commit.

Reply:
I’m reading the styling guide for Airbnb which has 80k stars, and they don’t mind skipping the block:

https://github.com/airbnb/javascript/blob/master/README.md#blocks--braces

But they also like cuddled elses, so I’m not sure if they can be trusted 😂